client/app/components/calendar/event/DeleteButton.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react';
+import axios from 'axios';
+import './event.scss';
+
+export default class DeleteButton extends React.Component {
+  static propTypes = {
+    id: PropTypes.string.isRequired
+  };
+
+  _path() {
+    return `/events/${this.props.id}`;
+  }
+
+  _processDelete(event) {
```

Comment:
We've been using the convention to name handlers as `handle[EventName]` e.g. `handleDelete`